### PR TITLE
Fapolicyd fixes

### PR DESCRIPTION
**Pull request by dsugar100**

### CODE REVIEW

File: policy/modules/admin/fapolicyd.if [hunk 1]
```diff
@@ -152,6 +152,8 @@ interface(`fapolicyd_admin',`
 	files_search_runtime($1)
 	admin_pattern($1, fapolicyd_runtime_t)
 
-	fapolicyd_run_fagenrules($1, $2)
+	ifndef(`direct_sysadm_daemon',`
+		fapolicyd_run_fagenrules($1, $2)
+	')
```
Comment (pebenito) [inline policy/modules/admin/fapolicyd.if:157]: Please move this to the end of the interface.

Comment (pebenito) [reply policy/modules/admin/fapolicyd.if:157]: Also, wouldn't this prevent the admin from generating rules on DIRECT_INITRC systems?

**Review by pebenito — CHANGES_REQUESTED** [review]
Please add a CI target that will test this build combination so we can check for this (and similar) problems.

File: policy/modules/admin/fapolicyd.if (hunk 1 above)
Comment (dsugar100) [reply policy/modules/admin/fapolicyd.if:157]: I'm not sure.  Is there a better way to solve this issue?  
in /lib/systemd/system/fapolicyd.service has an ExecStartPre that runs /usr/sbin/fagenrules - which is why there is init_script_domain(fagenrules_t, fagenrules_exec_t)
 in fapolicyd.te (and thus the conflict).

Comment (pebenito) [reply policy/modules/admin/fapolicyd.if:157]: Ok. I missed that.

**Comment by pebenito** [comment]
> I hope you wanted it in this same pull request.

Yes, thanks.

**Review by pebenito — APPROVED** [review]

**Pull request merged**